nitro-shim/Dockerfile

```diff
@@ -20,6 +20,9 @@ RUN amazon-linux-extras install aws-nitro-enclaves-cli -y && \
     rm -rf /var/cache/yum && \
     mkdir -p /enclave
 
+RUN yum install openssl -y
+RUN yum install libssl-dev openssl-devel -y
```

Comment:
Each instruction in a Dockerfile is a new layer, and the number of layers is better kept at a reasonable minimum. So there is not much reason to have these as two instructions. If readability is the concern, splitting it into multiple lines works just as well:

```Dockerfile
RUN yum -y install openssl \
    libssl-dev openssl-devel
```